### PR TITLE
Week1 - Assignments

### PR DESCRIPTION
**Pull request by 0xsagarthakur**
Topics are covered as part of Week 1 Assignments:

-  Intro to Web Fundamentals
- Browser Working & Mastering Git
- HTML Fundamentals

### CODE REVIEW

File: Week-1/Session-01/Assignment.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+    4. Build Quality Backlinks: Backlinks are a critical ranking factor for search engines. Develop a link building strategy to attract high-quality backlinks from authoritative websites relevant to your niche.
+
+    5. Ensure Mobile-Friendly Design: Optimize the website for mobile devices as mobile devices account for over half of all internet traffic. Use responsive web design, compress images, and ensure that the website's fonts are legible on mobile devices.
+
+    6. Leverage Social Media: Use social media channels to promote your website, engage with your audience, and build brand awareness. Social signals are an indirect ranking factor for search engines, and they can drive traffic to your website.
+
+    7. Regularly Monitor and Improve: Regularly monitor the website's performance in SERPs, and make necessary changes to improve the website's visibility and rankings. Use tools like Google Analytics, Google Search Console, and Ahrefs to track and analyze website performance.
+
+     > By following these steps, web developers can optimize their websites for better search engine rankings and drive more organic, non-paid traffic to their websites.
```
Comment (svashish305) [inline Week-1/Session-01/Assignment.md:108]: looks good to me

File: Week-1/Session-02/Assignments/Intermediate/HTML_Table.html [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+                <td>
+                    <h1>6</h1>
+                </td>
+            </tr>
+        </tbody>
+    </table>
+
+    <ol>
+        <li>Desktop</li>
+        <ul>
```
Comment (svashish305) [inline Week-1/Session-02/Assignments/Intermediate/HTML_Table.html:48]: Style should be circle as per question
<ul style="list-style-type:circle;">

File: Week-1/Session-02/Assignments/Intermediate/Hometown.html [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    <section>
+      <h2>Things to do</h2>
+      <ul>
+        <li>Visit the ancient Jain Temples</li>
+        <li>Trekking and Hiking</li>
+        <li>Boating</li>
+      </ul>
+    </section>
+  </body>
+</html>
```
Comment (svashish305) [inline Week-1/Session-02/Assignments/Intermediate/Hometown.html:58]: looks good

File: Week-1/Session-02/Assignments/Intermediate/Poem.html [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+    <p>
+        How dreary - to be - somebody!<br>
+        how public - like a Frog -<br>
+        To tell one's name - the livelong June -<br>
+        To an Admiring Bog!
+    </p>
+
+    <p>>
+        <cite>By Emily Dickinson(1891)</cite>
```
Comment (svashish305) [inline Week-1/Session-02/Assignments/Intermediate/Poem.html:27]:
```suggestion
        <cite>By Emily Dickinson (1891)</cite>
```

File: Week-1/Session-02/Assignments/Advanced/NewsArticle.html [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>5 Things to know about Rattlesnakes and their Babies</title>
+</head>
+
+<body>
+    <img src="./images/Universitylogo.png" alt="Logo of The University of Arizona">
+    <h1>5 Things to Know About Rattlesnakes and their Babies</h1>
+    <p>UA College of Pharmacy|Aug. 6,2014</p>
```
Comment (svashish305) [inline Week-1/Session-02/Assignments/Advanced/NewsArticle.html:14]:
```suggestion
    <p>UA College of Pharmacy | Aug. 6,2014</p>
```

File: Week-1/Session-03/DevTools_HandsOn.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+### 
+    ### Situation: You want to test how a webpage looks and behaves on different screen sizes.
+    Question: Which panel in Chrome DevTools should you use to test the responsiveness of a webpage on different screen sizes?
+    > Toggle Device Toolbar
+
+
+# Instructions
+
+### Go to chrome dev tools and analyze every panel and answer the question
+### Go to panel and play around according to each situation mentioned
```
Comment (svashish305) [inline Week-1/Session-03/DevTools_HandsOn.md:37]: looks good, where are rest of the files for session 3 assignment?

**Review by svashish305 — CHANGES_REQUESTED** [review]
address comments and re-request review once done

File: Week-1/Session-02/Assignments/Intermediate/HTML_Table.html (hunk 2 above)
Comment (0xsagarthakur) [reply Week-1/Session-02/Assignments/Intermediate/HTML_Table.html:48]: As I am seeing in the document nowhere is mentioned about the Styling.

image

File: Week-1/Session-02/Assignments/Intermediate/Poem.html (hunk 4 above)
Comment (0xsagarthakur) [reply Week-1/Session-02/Assignments/Intermediate/Poem.html:27]: Sure, I will update this suggested changes.

File: Week-1/Session-02/Assignments/Advanced/NewsArticle.html (hunk 5 above)
Comment (0xsagarthakur) [reply Week-1/Session-02/Assignments/Advanced/NewsArticle.html:14]: Sure, I will update these suggested changes.

File: Week-1/Session-03/DevTools_HandsOn.md (hunk 6 above)
Comment (0xsagarthakur) [reply Week-1/Session-03/DevTools_HandsOn.md:37]: These are the instructions to follow to give the answer to the above questions in the file.

File: Week-1/Session-02/Assignments/Intermediate/HTML_Table.html (hunk 2 above)
Comment (svashish305) [reply Week-1/Session-02/Assignments/Intermediate/HTML_Table.html:48]: no but look at the unordered list here, default is of type disc which is solid black filled bullet point
but here its circle.

Comment (0xsagarthakur) [reply Week-1/Session-02/Assignments/Intermediate/HTML_Table.html:48]: Correct but when I opened my HTML file in the browser the ul list comes with a circle only

image

Comment (svashish305) [reply Week-1/Session-02/Assignments/Intermediate/HTML_Table.html:48]: turns out default style type for ul inside ol is circle, not disc.

**Review by svashish305 — APPROVED** [review]
good work

**Pull request merged**